src/OpaqueVal.h

```diff
@@ -132,28 +119,11 @@ class OpaqueVal : public Val {
     explicit OpaqueVal(OpaqueTypePtr t);
     ~OpaqueVal() override = default;
 
-    /**
-     * Serializes the value into a Broker representation.
-     *
-     * @return the broker representation, or an error if serialization
-     * isn't supported or failed.
-     */
-    [[deprecated("Remove in v7.1: use SerializeData instead")]] broker::expected<broker::data> Serialize() const;
-
```

Comment:
Think we should call this one out again in the NEWS. There's already a NEWS entry for when `DoSerialize()` and `DoUnserialize() was deprecated, but the caveat was that plugin authors won't get compiler warnings.

With these now removed, plugins with OpaqueVal implementations will just fail to compile, so might be good to pointing that out again.